src/test/components/Timeline.test.js

```diff
@@ -240,6 +241,61 @@ describe('Timeline multiple thread selection', function() {
       '  - show [thread Style]',
     ]);
   });
+
+  it('will select a thread through enter and spacebar keypresses', function() {
```

Comment:
Thanks for the new tests!

Reply:
Note: [according to codecov](https://codecov.io/gh/firefox-devtools/profiler/compare/12ccc8e076c1ab2ddbb6bcd2e9ad109ea9a7abd0...45a1d7bbc9437ff691fc02511e11ea6f3ef57f19/diff), the path with `Enter` isn't tested for GlobalTracks.

Reply:
Ah, I can duplicate the tests and make a global and local variant.